examples/bert/bert_cuda_gpu.json

```diff
@@ -59,7 +59,7 @@
         "perf_tuning": {
             "type": "OrtPerfTuning",
             "config": {
-                "io_bind": true
+                "enable_cuda_graph": true
```

Comment:
If user do not set this value, will it be tuned even if the EP is cuda.

Reply:
It is better to let user who has cuda_graph knowledge make the choise. 
If user doesn't set enable_cuda_graph, it will not be tuned.